contrib/systemd/torus.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Torus daemon
+Documentation=https://www.torus.sh/docs/
+After=network.target
+
+[Service]
+Type=simple
+User=torus
+Environment="TORUS_ROOT=/var/run/torus"
+EnvironmentFile=/etc/torus/token.environment
+ExecStart=/usr/bin/torus daemon start --foreground --no-permission-check
```

Comment:
So in "service mode" anyone on the system gets access to any of the secrets that the systems identity provides it?